packages/react-components/src/components/Picker/PickerList.tsx

```diff
@@ -32,17 +32,16 @@ export const PickerList: React.FC<IPickerListProps> = ({
   onClose,
   onSelect,
 }) => {
-  const mergedClassNames = cx(
-    baseClass,
-    `${baseClass}--${size}`
-  );
+  /* eslint-disable */
+  const mergedClassNames = cx(baseClass, `${baseClass}--${size}`);
+  /* eslint-enable */
```

Comment:
I've disabled it because I get:
`35:59  error  Invalid type "any" of template literal expression  @typescript-eslint/restrict-template-expressions`

I don't know why, because we have the same implementation in `Trigger.tsx`